HISTORY.rst

```diff
@@ -3,6 +3,8 @@ History
 =======
 
 0.1.0 (2018-03-28)
+0.2.0 (2018-06-12)
 ------------------
 
 * First release on PyPI.
+* Second release on PyPI.
```

Comment:
It's our first release on PyPI.

Reply:
Should I remove the two lines about 0.1.0?

Reply:
Oh, I'm not sure. Maybe we should just call this release 0.1.0 after all.

Reply:
All the text that is there now is from when I started the project by using template-generating code called cookiecutter: https://github.com/audreyr/cookiecutter

So no need to take the files generated by cookiecutter too seriously. They're not particularly customized to our project.